src/elements/content-explorer/DeleteConfirmationDialog.scss

```diff
@@ -0,0 +1,5 @@
+.be-modal.be-modal-delete {
+    .be-modal-dialog-content {
```

Comment:
I think we'll eventually want to upgrade the elements to use our internal modal. But this is consistent to how we're doing the other modals in the elements so I think we're good to keep.